daisy_workflows/build-publish/rhui/health_check.py

```diff
@@ -0,0 +1,131 @@
+#  Copyright 2022 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Health check for RHUI infrastructure nodes."""
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+import typing
+
+
+class HealthChecks:
+  nfs_mount: str
+
+  def __init__(
+      self,
+      nfs_mount: str,
+  ) -> None:
+    self.nfs_mount = nfs_mount
+
+  def mount_uses_nfs(self, logger: logging.Logger):
+    logger.info("checking that %s is mounted as NFS", self.nfs_mount)
+    # Using pipes to capture output since RHEL8 has Python 3.6
+    # which doesn't support `capture_output`.
+    p = subprocess.run(
+      ["stat", "-fc", "%T", self.nfs_mount],
```

Comment:
it's a file, we really don't need to shell out to check it, python has all the tools to do it. if you want to kick this off now like this it's fine but i'd rather it just do things in a python directly

Reply:
sgtm; changed to read `/proc/mounts`